bin/setup

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env ruby
-require 'pathname'
 require 'fileutils'
 include FileUtils
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+APP_ROOT = File.expand_path('..', __dir__)
```

Comment:
I mean ok, but why? Or are you just conforming to what Rails 5 now generates?

Reply:
I ran `rake app:update` and kept all the changes to the default generated file changes